packages/viem-account-ledger/src/ledger-to-account.ts

```diff
@@ -0,0 +1,66 @@
+import { CELO_DERIVATION_PATH_BASE, trimLeading0x } from '@celo/base'
+import { ensureLeading0x } from '@celo/utils/lib/address.js'
```

Comment:
this is exported from @celo/base (also address.js) and i think would eliminate need for utils as dependency